### ISSUE TITLE
Incorrect photometric distortions in imagenet example

### ISSUE THREAD

**Issue by saurabhme** [vlfeat/matconvnet#613]
Code uses `net.meta.augmentation.rgbVariance = 0.1*sqrt(d)*v' ;` in line [69 of cnn_imagenet.m](https://github.com/vlfeat/matconvnet/blob/master/examples/imagenet/cnn_imagenet.m#L69) . It should be `0.1*v*sqrt(d)` instead.

**Comment by vedaldi** [comment]
Hi, thanks that's correct. We recently found this bug and it will be corrected in the next version.

**Comment by saurabhme** [comment]
Sounds good!

**Comment by jstudy** [comment]
@saurabhme  Hi, how much does this error affect the final performance? Have you tested it ?

**Comment by saurabhme** [comment]
Nope, i haven't trained a full model with it fixed. I don't think it would make much difference as the buggy code still adds reasonable randomness anyway.
